resource_obmcs_objectstorage_object.go

```diff
@@ -8,10 +8,69 @@ import (
 	"github.com/hashicorp/terraform/helper/schema"
 	"github.com/oracle/bmcs-go-sdk"
 
+	"crypto/md5"
+	"encoding/hex"
+	"strings"
+
 	"github.com/oracle/terraform-provider-oci/crud"
 )
 
 func ObjectResource() *schema.Resource {
+	var objectSchema = map[string]*schema.Schema{
+		"namespace": {
+			Type:     schema.TypeString,
+			Required: true,
+			Computed: false,
+		},
+		"bucket": {
+			Type:     schema.TypeString,
+			Required: true,
+			Computed: false,
+		},
+		"object": {
+			Type:     schema.TypeString,
+			Required: true,
+			Computed: false,
+		},
+		"content": {
+			Type:     schema.TypeString,
+			Optional: true,
+			StateFunc: func(body interface{}) string {
```

Comment:
So this will no longer write the content to the state file, correct? Will this be an issue for users who upgrade?

Reply:
It will cause the same object to be put again

Reply:
Ok, sounds reasonable to have a one time update on upgrade, given that we don't really support large files.